src/gluonts/distribution/lds.py

```diff
@@ -315,7 +317,10 @@ def sample(
         # Sample the prior state.
         # samples_lat_state: (num_samples, batch_size, latent_dim, 1)
         state = MultivariateGaussian(
-            self.prior_mean, F.linalg_potrf(self.prior_cov)
+            self.prior_mean,
+            jitter_cholesky(
+                F, self.prior_cov, self.latent_dim, float_type=np.float32
+            ),
```

Comment:
Could this be made optional? Like having `sampling_jitter=False` in the constructor, and set `sampling_jitter=True` when constructing the distribution in `DeepStateEstimator`?

Reply:
Ah.. forgot to add a comment about this. The `jitter_cholesky` method applies jitter only if the Cholesky decomposition fails for the given matrix and returns normal result without jitter if there is no problem. So this is essentially having the jitter only for problematic cases (btw, this happened only for the constant dataset in the evaluations).

I think `jitter_cholesky` was written this way to avoid having further option/flag like `sampling_jitter` on the caller side.

Reply:
Yes exactly. It only applies the jitter when it is needed and otherwise does the standard Cholesky decomposition.